Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -0,0 +1,319 @@
+import base64
+from datetime import datetime
+import dateutil.parser
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+from typing import Any, Dict, Union, Optional, Tuple, List
+
+FILTER_RELATIONSHIP_AND = "FILTER_RELATIONSHIP_AND"
+
+DEMISTO_OCCURRED_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+RECO_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+DEMISTO_INFORMATIONAL = 0.5
+RECO_API_TIMEOUT = 30  # Increase timeout for RECO API
+RECO_ACTIVE_INCIDENTS_VIEW = "active_incidents_view"
+RECO_INCIDENT_ID_FIELD = "incident_id"
+CREATED_AT_FIELD = "created_at"
+STEP_FETCH = "fetch"
+STEP_INIT = "init"
+
+
+class RecoClient:
+    def __init__(self, client: BaseClient):
+        self.client = client
+
+    def get_incidents(
+            self,
+            risk_level: Optional[int] = None,
+            source: Optional[str] = None,
+            before: Optional[datetime] = None,
+            after: Optional[datetime] = None,
+            limit: int = 1000,
+    ) -> List[Dict[str, Any]]:
```

Comment:
Can't we import the generated types here, instead of any? We have this API declaratively in our autogenerated code from swagger (including, for example, the request type, the URLs, etc. etc.).